### PR TITLE
added lexer and recognizer files

### PR DESCRIPTION
**Pull request by jfinlayson78**

### CODE REVIEW

File: lexer_valid_tokens.kxi [hunk 1]
```diff
@@ -0,0 +1,608 @@
+// if you interpret this as C in vsCode it's easier to work with
+
+// Keywords
+bool 
+break
+--case
```
Comment (jbeBan) [inline lexer_valid_tokens.kxi:6]: Is the `--` before `case` necessary?

File: lexer_valid_tokens.kxi [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+
+// Keywords
+bool 
+break
+--case 
+class 
+char 
+cin 
+cout 
+--default
```
Comment (jbeBan) [inline lexer_valid_tokens.kxi:11]: Is the `--` before `default` necessary?

File: lexer_valid_tokens.kxi [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+for 
+if 
+int 
+new 
+null 
+public 
+private 
+return 
+string 
+--switch
```
Comment (jbeBan) [inline lexer_valid_tokens.kxi:23]: Is the `--` before `switch` necessary?

File: lexer_valid_tokens.kxi (hunk 1 above)
Comment (jfinlayson78) [reply lexer_valid_tokens.kxi]: resolved

File: lexer_valid_tokens.kxi (hunk 2 above)
Comment (jfinlayson78) [reply lexer_valid_tokens.kxi]: resolved

File: lexer_valid_tokens.kxi (hunk 3 above)
Comment (jfinlayson78) [reply lexer_valid_tokens.kxi]: resolved

**Review by jbeBan — APPROVED** [review]

**Pull request merged**